.github/workflows/ci.yml

```diff
@@ -43,6 +55,8 @@ jobs:
           override: true
       - name: Install cargo-llvm-cov
         uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Test in release
+        run: cargo test --features multicore --benches --workspace --release
```

Comment:
I didn't know about the `--benches` flag to `cargo test`; what does it actually do? I see out put like 

```
Testing Presets/(U128) Random safe prime
Success
```

Does it actually run the bench one time? Or is it a pure compile time check?

Reply:
Runs it once, as far as I understand.

I only learned about it recently myself; I kind of wanted that functionality for a while but never really looked into it.